cmd/scw/main.go

```diff
@@ -28,7 +30,15 @@ var (
 	GoArch    = runtime.GOARCH
 )
 
+func cleanup() {
+	if r := recover(); r != nil {
+		fmt.Println(sentry.ErrorBanner)
+		fmt.Println("stacktrace from panic: \n" + string(debug.Stack()))
+	}
+}
```

Comment:
Consider moving 
```
if buildInfo.IsRelease() {
     sentry.RecoverPanicAndSendReport(buildInfo)
}
```

In this defer func.

This would require cleanup to take buildInfo as parameter